src/modules/songs/forms/AddSongToSet/components/AddSongToSetDialog.tsx

```
@@ -0,0 +1,131 @@
+      <DialogContent
+        fixed
+        minimalPadding
+        closeButton={null}
+        className="max-h-[90%] gap-0 overflow-y-auto p-0 lg:max-h-[65%] lg:p-0"
+      >
+        {/* TODO: move within each step's component */}
+        <AddSongToSetDialogHeader
+          title={
+            currentStep === AddSongToSetDialogStep.SELECT_SET &&
+            isCreatingNewSet
+              ? contentMap[AddSongToSetDialogStep.SELECT_SET].alternateTitle! // using a non-null type assertion since we know this specific alternateTitle shouldn't be undefined
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)